lib/general.sh

```diff
@@ -718,6 +722,16 @@ prepare_host()
 	fi
 }
 
+# downloader() <url> <outputfilename>
+downloader()
+{
+	[ $# -ne 2 ] && exit_with_error "downloader args count" "$#"
+	args='--file-allocation=falloc --auto-file-renaming=false --continue=true --allow-overwrite=false'
+	display_alert "downloader args 1" "$1" "info"
+	display_alert "downloader args 2" "$2" "info"
+	aria2c $args "$1" -o "$2"
```

Comment:
Perhaps we could run aria2c with: --console-log-level=error --download-result=hide